### PR TITLE
Migrate weblogo3 from devteam + condafy

### PR DESCRIPTION
**Pull request by abretaud**
Migrated from https://github.com/galaxyproject/tools-devteam/tree/master/tools/weblogo3

### CODE REVIEW

File: tools/weblogo3/rgWebLogo3.xml [hunk 1]
```diff
@@ -0,0 +1,133 @@
+<tool id="rgweblogo3" name="Sequence Logo" version="0.5">
```
Comment (bgruening) [inline tools/weblogo3/rgWebLogo3.xml]: Should we increase the tool version there to 3.5.0?

File: tools/weblogo3/rgWebLogo3.xml [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    <description>generator for fasta (eg Clustal alignments)</description>
+    <requirements>
+        <requirement type="package" version="3.5.0">weblogo</requirement>
+    </requirements>
+    <command><![CDATA[
+    python '$__tool_directory__/rgWebLogo3.py' -F $outformat -s $size -i '$input' -o '$output' -t '$logoname' -c '$colours' -U '$units'
```
Comment (bgruening) [inline tools/weblogo3/rgWebLogo3.xml]: every parameter on its own line?

File: tools/weblogo3/rgWebLogo3.py [hunk 3]
```diff
@@ -0,0 +1,161 @@
+"""
```
Comment (bgruening) [inline tools/weblogo3/rgWebLogo3.py]: I guess we can remove this.

File: tools/weblogo3/rgWebLogo3.py [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+            self.clparams['-c'] = self.opts.colours
+        if self.opts.units is not None:
+            self.clparams['-U'] = self.opts.units
+        s = self.runCL()
+        return check, s
+
+
+if __name__ == '__main__':
+    '''
+    called as
```
Comment (bgruening) [inline tools/weblogo3/rgWebLogo3.py]: This can also be remove I think.

File: tools/weblogo3/rgWebLogo3.py [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+        tlogs = ''.join(open(templog, 'r').readlines())
+        if len(tlogs) > 1:
+            s = '## executing %s returned status %d and log (stdout/stderr) records: \n%s\n' % (cl, rval, tlogs)
+        else:
+            s = '## executing %s returned status %d. Nothing appeared on stderr/stdout\n' % (cl, rval)
+        os.unlink(templog)  # always
+        if rval != 0:
+            print('## rgWebLogo3.py error - executing %s returned error code %d' % (cl, rval), file=sys.stderr)
+            print('## This may be a data problem or a tool dependency (%s) installation problem' % WEBLOGO, file=sys.stderr)
+            print('## Please ensure %s is correctly installed and working on the command line -see http://code.google.com/p/weblogo' % WEBLOGO, file=sys.stderr)
```
Comment (yhoogstrate) [inline tools/weblogo3/rgWebLogo3.py]: http://code.google.com/p/weblogo -> https://github.com/weblogo/weblogo

**Comment by bgruening** [comment]
Thanks!!!

**Comment by martenson** [comment]
IUC has been given admin permissions on this repo at TTS and MTS

**Pull request merged**